src/pages/demo/staffProfiles/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { drupalClient } from '@/utils/drupalClient'
+import {
+  StaffProfiles,
+  StaffNewsProfiles,
+} from '@/components/paragraph/staff_profile'
+import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
+import Container from '@/components/container'
+
+const StaffProfilePage = ({ nodes }) => {
+  if (!nodes) return
+  return (
+    <>
+      <Container className="container">
```

Comment:
I think it would be good to provide a divider between each iteration, just to be clear where a pair begins and ends. An `<hr />` would be fine.